BUILD.plz

```diff
@@ -4,6 +4,22 @@ github_repo(
     revision = "eabddd60c960f47dac736c02602123ae6e210dd2",
 )
 
+genrule(
+    name = "go.mod",
+    outs = ["go.mod"],
+    cmd = [
+        """cat > $OUTS << EOF
+module plz-out
+
+go 1.14
```

Comment:
**Question**: if this is just a dummy module, can we consider deriving a version here from a source instead of hard-coding one to avoid maintaining this version? 
Such maintenance is usually unnecessary for a dummy module, but can still cause confusion and edge case issues when a version go out of support or a non-backward compatible version is promoted to default.

Reply:
Well, it doesn't matter since this go.mod file won't ever be used, but for the sake of consistency: done.